include/swift/AST/DiagnosticsSema.def

```diff
@@ -7390,7 +7390,7 @@ ERROR(init_accessor_can_refer_only_to_properties,none,
       (DescriptiveDeclKind, DeclNameRef))
 
 ERROR(init_accessor_initializes_attribute_on_other_declaration,none,
-      "initalizes(...) attribute could only be used with init accessors",
+      "initializes(...) attribute could only be used with init accessors",
```

Comment:
Looks like there is no test case that exercise this diagnostic.
I think is fine to take this PR as is, but if you up to it, can you try to figure out a test the exercise this and add it to the suit?

Reply:
All diagnostics emitted by the compiler ideally should have a test case in the test suit where diagnostic verifier ensures that a specific incorrect code produces that expected diagnostic. Here is an example of a few attribute diagnostics: https://github.com/apple/swift/blob/main/test/attr/attributes.swift

In this case diagnostic message is defined and emitted but there is no occurrences of it in test suit, which means there is no test case exercising it. I see few in swift syntax directory, but not in the diagnostic verifier, so if you up to add it in this PR or a follow up. There are docs on how to modify and run tests locally in this section https://github.com/apple/swift/blob/main/docs/HowToGuides/GettingStarted.md#running-tests =]

Reply:
We need to remove both `initializes` and `accesses` diagnostic definitions because we have recently moved to a different syntax to express this.

Reply:
Ah I see, there is also code emitting this diagnostic as well, so we would need to remove this also. And I saw some swift syntax test cases as well... that is why I thought this would still valid and just missing tests.

Reply:
Ah, alright then. Is there anything I can do here, or would you like me to close my pull request if you prefer.

Reply:
I think I saw @ahoppen removing `initializes` and `accesses` support from swift-syntax in one of the recent PRs but I might be mistaken. We can just take this PR as is look into removing the code separately, there is no harm is fixing a typo.

Reply:
Yes, I did remove it but I can’t find my PR right now. We should also remove the code from the C++ parser but I agree there’s no harm in fixing the typo here since removing support for `initializes` and `accesses` is a separate issue.

Reply:
@xedin @ahoppen It is safe to remove the initializes and access attribute code as well? Or they have to be keep for compatibility reasons(specially from serialization)? I'm no following the project as much lately so asking to be sure =]

Reply:
It's safe to remove the code because these attributes were never actually used in anything.

Reply:
Thank you @xedin =]
PR is there https://github.com/apple/swift/pull/68073